src/components/TextInput/index.tsx

```diff
@@ -40,6 +42,7 @@ export default function TextInput({
         $error={errorText !== ''}
         placeholder={placeholder}
         id={id}
+        height={height}
```

Comment:
should we remove this `height` prop?

Reply:
deferring to @ccatherinetan here bc i think she added it!

Reply:
yeah i ran typescript compiler and everything seems chill ... got rid of it